### ISSUE TITLE
Do not override default behavior for dropdown

### ISSUE THREAD

**Issue by jnarowski** [creativetimofficial/vue-light-bootstrap-dashboard-pro#2]
Another issue, the default behavior for button dropdowns seem to be hijacked with your custom drop down. Can you namespace this so it only affects the custom plugin you developed. The default bootstrap behavior for a dropdown is now broken. Then menu hovers strangely and doesn't appear as expected. 

Anytime I want to add a dropdown even with the vanilla bootstrap code, it doesn't work as expected.

**Comment by alexandru-paduraru** [comment]
@jnarowski thank you for pointing this out, this seems to be a problem that is coming from our original Light Bootstrap Dashboard PRO (https://www.creative-tim.com/product/light-bootstrap-dashboard-pro). We will fix it there and then fix it in Vue too.

In the meantime, you can send us a link to the issue and we will give a quick solution.

Best,
Alex

**Comment by cristijora** [comment]
Temporary solution until the new version is released.
1. Move the code from `assets/sass/lbd/_dropdown.scss` under a 
`<style scoped lang="scss">` tag inside `Dropdown.vue`
2. Add these 2 lines before the css code
```css
 @import "~assets/sass/lbd/variables";
 @import "~assets/sass/lbd/mixins";
```

3 Remove `@import "lbd/dropdown";` line 41 from `light-bootstrap-dashboard.scss`

This will make the style for dropdowns scoped to the Dropdown.vue component and won't override any bootstrap styles/behavior.

**Comment by cristijora** [comment]
Fixed in 1.1.0 release. See [Changelog](https://github.com/creativetimofficial/vue-light-bootstrap-dashboard-pro/blob/master/CHANGELOG.md) and [migration guide](http://vuejs.creative-tim.com/vue-light-bootstrap-dashboard-pro/documentation/#/migration_guide). To get the new version go to [Downloads section](https://www.creative-tim.com/downloads) and redownload the product.
